.github/workflows/e2e.yml

```diff
@@ -32,7 +32,3 @@ jobs:
         run: make -j2 docker runner tests
         if: "env.GIT_DIFF != ''"
 
-      - name: Run CI testnets
-        working-directory: test/e2e
-        run: ./run-multiple.sh networks/ci.toml networks/propagation-reactor-compatibility.toml
-        if: "env.GIT_DIFF != ''"
```

Comment:
How is this related to legacy gossip? Do e2e tests fail without legacy gossip?

Reply:
great catch, I only wanted to delete the compatibility test, not all e2e

Reply:
7a7799a8e19ae001f1b3348c93a76d230b665d96